test/jdk/TEST.groups

```diff
@@ -200,6 +200,8 @@ jdk_time = \
 
 jdk_rmi = \
     java/rmi \
+    -java/rmi/registry/nonLocalRegistry/NonLocalRegistryTest.java \
+    -java/rmi/registry/nonLocalRegistry/NonLocalSkeletonTest.java \
```

Comment:
RunTests.gmk runs jtreg with -automatic so I don't think you need to exclude the manual tests from these test groups.